services/app-api/prisma/migrations/20220208221835_add_revisions_table/migration.sql

```diff
@@ -0,0 +1,19 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- CreateTable
+CREATE TABLE "StateSubmissionRevision" (
+    "id" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL,
+    "submissionID" TEXT NOT NULL,
+    "submissionFormProto" BYTEA NOT NULL,
+
+    CONSTRAINT "StateSubmissionRevision_pkey" PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "StateSubmissionRevision" ADD CONSTRAINT "StateSubmissionRevision_submissionID_fkey" FOREIGN KEY ("submissionID") REFERENCES "StateSubmission"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
+
+-- CreateRevisionForEachExistingSubmission
+INSERT INTO "StateSubmissionRevision" 
+SELECT uuid_generate_v4() AS "id", now() AS "createdAt", "id" AS "submissionID","submissionFormProto"
+FROM "StateSubmission";
```

Comment:
Once we’ve done some of the API work, we should drop the submissionFormProto column from the old StateSubmission table.